src/sgb-v/segments.ts

```diff
@@ -0,0 +1,7 @@
+import { FileType } from "../types";
+import { mask, number, price, day, month, date, time, datetime, segment } from "../formatter";
+
+const Syntax_Version = "UNOC:3";
```

Comment:
UNOC definiert dass alle Zeichen in dem Interchange in ISO 8859-1 (also nicht UTF-8) kodiert sind. Das könnte zu Problemen bei Sonderzeichen führen. Weiß nicht ob es eine einfache Art gibt, in JS Strings um-zukodieren.

Copy&paste aus der Dokumentation:

Syntax identifier | ISO standard | Languages
-- | -- | --
UNOA | 646 |  
UNOB | 646 |  
UNOC | 8859 - 1 | Danish, Dutch, English, Faeroese,   Finnish, French, German, Icelandic, Irish, Italian, Norwegian, Portuguese,   Spanish, Swedish
UNOD | 8859 - 2 | Albanian, Czech, English, Hungarian,   Polish, Romanian, Serbo-Croatian, Slovak, Slovene
UNOE | 8859 - 5 | Bulgarian, Byelorussian, English, Macedonian,   Russian, Serbo-Croatian, Ukrainian
UNOF | 8859 - 7 | Greek

Reply:
Interesting finding. 
Transcoding could be possible using the TextEncoder / TextDecoder API. 

I suppose transcoding from ISO-8859-1 to UTF-8 is not a problem, but the other way for sure. What will happen to special turkish or polish charaters in names? We need to test that. I suggest we open an issue and come back to this later.

Reply:
Special Turkish or Polish characters are not possible in ISO 8859-1.

So, if you want to invoice a health care service at a German health insurance and you have a Turkish or Polish name, ... game over ;-)

Seriously, I guess this issue must have come up before and this is something that maybe Alex could also enquire at the public contact. My guess is that they "solved" this by only letting you register names with ISO 8859-1 and other characters need to be rewritten (ą -> a etc). But let's see! Will you relay this to him?

Reply:
ok

Reply:
I'll resolve conversation because you made a ticket for that